docs/source/chainermn/installation/guide.rst

```diff
@@ -121,11 +125,22 @@ We tested ChainerMN on all the following environments.
 * Python 2.7.13, 3.5.1, 3.6.1
 * MPI
 
-  * openmpi 2.1.2
+  * Open MPI 2.1.6, 3.0.4, 3.1.4
 
 * MPI4py 3.0.0
 * NCCL 2.3.2 2.4.2
 
+.. note::
+
+  Note that certain versions of Open MPI have a bug that might cause ChainerMN program to hang.
```

Comment:
Note that the **following** versions of ....

Reply:
Fixed! Thanks!